About.scss

```diff
@@ -0,0 +1,233 @@
+@import "variables.scss";
+
+.About {
+  background-repeat: no-repeat;
+  background-size: 110%;
+  margin-bottom: calc(#{$padding}*2) !important;
+  @include desktop() {
+    padding-top: calc(#{$padding} * 4) !important;
+    background-size: 165%;
+  }
+  @include xl() {
+    background-size: 100%;
+  }
+
+  .margin {
+    @include desktop() {
+      margin-top: calc(#{$padding} * 4) !important;
+    }
+  }
+
+  h1 { color: $teal; font-size: 24px; line-height: 32px; padding: calc(#{$padding}*3) calc(#{$padding}*3) 0 calc(#{$padding}*3);
```

Comment:
This is chiell, though normally I end up regretting it when I apply colors and stuff top-ish level based on the tag alone, since e.g any other `h1` that winds up in this component will get hit with `$teal` as well. In the future we can put stuff like different colors into their own classes, so `.teal { color: $teal }` would be a thing at a global level and then you can just toss the class on any elements that need it. But yea, font sizing and stuff makes lots of sense here, noice jerb 👍